modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -387,12 +387,37 @@ definition {
 	}
 
 	@description = "Verify that the user is able to Reverse Sort Direction by clicking the icon"
-	@ignore = "Test Stub"
 	@priority = "2"
 	test CanReverseSortDirection {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
```

Comment:
In this `.testcase` file, the `setUp` already assign Blogs to **single approver**, Please remove this piece. Thank you!